service/src/test/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreatePostgresqlDbStepTest.java

```diff
@@ -136,13 +135,24 @@ void doStepSuccess() throws InterruptedException {
     when(mockParametersResolver.getValue(
             CromwellBaseResourcesFactory.ParametersNames.POSTGRES_SERVER_STORAGE_SIZE_GB.name()))
         .thenReturn(storageSize);
+    when(mockParametersResolver.getValue(
+            CromwellBaseResourcesFactory.ParametersNames.ENABLE_PGBOUNCER.name()))
+        .thenReturn("true");
 
     StepResult stepResult = createPostgresqlDbStep.doStep(mockFlightContext);
 
     assertThat(stepResult.getStepStatus(), equalTo(StepStatus.STEP_RESULT_SUCCESS));
 
     verifyServerProperties(postgresqlSku, skuTier, backupRetention, storageSize, serverVersion);
     verifyBasicTags(postgresqlTagsCaptor.getValue(), LANDING_ZONE_ID);
+    // This assertion cannot go in veryBasicTags as it only applies to PostgresqlDb
```

Comment:
heh those tags are very basic!! Did you mean verifyBasicTags? :)

Reply:
ha yes I did :)